examples/7_qpt_circuit.jl

```diff
@@ -38,6 +38,7 @@ U = tomography(data, U0;
                optimizer = SGD(η = 0.1),
                batchsize = 500,
                epochs = 5,
+               tp_regularizer = 0.1,
```

Comment:
Would be better to explicitly say `trace_preserving_regularizer`.

Reply:
OK